src/game/client/displaystring.h

```diff
@@ -39,7 +39,7 @@ class DisplayString : public MemoryPoolObject
     virtual void Set_Font(GameFont *font) { m_font = font; }
     virtual GameFont *Get_Font() { return m_font; }
     virtual void Set_Word_Wrap(int length) = 0;
-    virtual void Set_Word_Wrap_Centered(char unk) = 0;
+    virtual void Set_Word_Wrap_Centered(bool unk) = 0;
```

Comment:
name the arg then, state/warp_centered/on

Reply:
fixed.